cumulus_library/template_sql/codeable_concept_denormalize.sql.jinja

```diff
@@ -120,13 +120,12 @@ CREATE TABLE {{ target_table }} AS (
             ROW_NUMBER()
                 OVER (
                     PARTITION BY id
-                    ORDER BY priority ASC
+                    ORDER BY priority ASC, code ASC
                 ) AS available_priority
         FROM union_table
         GROUP BY
             id, row, priority, system, code, display, userSelected
             {%- for extra_field in extra_fields %}, {{ extra_field[1] }}{% endfor %}
-        ORDER BY priority ASC
```

Comment:
AFAICT this is not doing anything here - the next bit just pulls a single line from this temp table, based on `available_priority`, so we don't need to sort this.

Reply:
probably a debugging convenience that hung around